tests/FINDOLOGIC/Export/Tests/PropertyTest.php

```diff
@@ -71,12 +69,15 @@ public function testReservedPropertyKeysCausesException(string $key, bool $shoul
 
     public function testNonAssociativePropertyValueCausesException(): void
     {
-        try {
-            new Property('foo', ['bar']);
-        } catch (Exception $exception) {
-            $warningMessage = 'Property values have to be associative, like $key => $value. The key "0" has to be a ' .
-                'string, integer given.';
-            $this->assertEquals($exception->getMessage(), $warningMessage);
-        }
+        $warningMessage = 'Property values have to be associative, like $key => $value. The key "0" has to be a ' .
+            'string, integer given.';
+
+        set_error_handler(static function (int $errno, string $error): never {
```

Comment:
:question: Is this error handler removed after this test is executed? Otherwise, it could stick around and pollute other tests.

Reply:
:thinking: It's probably legitimate to configure `convertWarningsToExceptions="true"` so we don't have to test warnings in such an awkward way.

Reply:
The completely changed the handling within PHPUnit 10, that's why I even needed to change that. This configuration does not exist anymore.

I added `restore_error_handler()` at the end of the test